src/test/accessibility/accessibility-admin-test.ts

```diff
@@ -0,0 +1,251 @@
+    I.fillField('#search-input', locationName);
+    I.selectOption('#listType', listType);
+
+    // Set a specific content date of 12/31/2020 so the request to Courtel can be identified.
+    const contentDate = new Date(2020, 11, 31);
+    I.fillField('#content-date-from-day', padFormatted(contentDate.getDate()));
+    I.fillField('#content-date-from-month', padFormatted(contentDate.getMonth() + 1));
+    I.fillField('#content-date-from-year', contentDate.getFullYear());
+    I.selectOption('#classification', 'Public');
+    I.fillField('#display-date-from-day', padFormatted(date.getDate()));
+    I.fillField('#display-date-from-month', padFormatted(date.getMonth() + 1));
+    I.fillField('#display-date-from-year', date.getFullYear());
+    I.fillField('#display-date-to-day', padFormatted(dayAfter.getDate()));
+    I.fillField('#display-date-to-month', padFormatted(dayAfter.getMonth() + 1));
+    I.fillField('#display-date-to-year', dayAfter.getFullYear());
+    I.click('Continue');
+    I.waitForText('Check upload details');
+    I.checkA11y('manual-upload-summary-a11y-audit.html');
+});
+
+Scenario('Admin User Journey - manual-upload-confirmation-page', async ({ I }) => {
+    I.amOnPage('/manual-upload');
+    I.waitForText('Manual upload');
+    I.attachFile('#manual-file-upload', '../end-to-end/shared/mocks/' + fileName);
+    I.fillField('#search-input', locationName);
+    I.selectOption('#listType', listType);
+
+    // Set a specific content date of 12/31/2020 so the request to Courtel can be identified.
+    const contentDate = new Date(2020, 11, 31);
+    I.fillField('#content-date-from-day', padFormatted(contentDate.getDate()));
+    I.fillField('#content-date-from-month', padFormatted(contentDate.getMonth() + 1));
+    I.fillField('#content-date-from-year', contentDate.getFullYear());
+    I.selectOption('#classification', 'Public');
+    I.fillField('#display-date-from-day', padFormatted(date.getDate()));
+    I.fillField('#display-date-from-month', padFormatted(date.getMonth() + 1));
+    I.fillField('#display-date-from-year', date.getFullYear());
+    I.fillField('#display-date-to-day', padFormatted(dayAfter.getDate()));
+    I.fillField('#display-date-to-month', padFormatted(dayAfter.getMonth() + 1));
+    I.fillField('#display-date-to-year', dayAfter.getFullYear());
+    I.click('Continue');
+    I.waitForText('Check upload details');
+    I.click('Confirm');
+    I.waitForText('Success');
+    I.checkA11y('manual-upload-confirmation-a11y-audit.html');
+});
+
+Scenario('Admin User Journey - manual-upload-confirmation-error-page', async ({ I }) => {
+    I.amOnPage('/manual-upload');
+    I.waitForText('Manual upload');
+    I.attachFile('#manual-file-upload', '../end-to-end/shared/mocks/' + invalidFileName);
+    I.fillField('#search-input', locationName);
+    I.selectOption('#listType', listType);
+
+    // Set a specific content date of 12/31/2020 so the request to Courtel can be identified.
```

Comment:
I do not believe this comment / same comments below are needed and therefore can be removed

Reply:
Removed, thanks:)